legal-api/src/legal_api/models/filing.py

```diff
@@ -688,6 +688,17 @@ def get_filings_by_types(business_id: int, filing_types):
             all()
         return filings
 
+    @staticmethod
+    def get_incomplete_filings_by_types(business_id: int, filing_types: list):
+        """Return the filings of particular types and statuses."""
+        filings = db.session.query(Filing). \
+            filter(Filing.business_id == business_id). \
+            filter(Filing._filing_type.in_(filing_types)). \
+            filter(Filing._status != Filing.Status.COMPLETED.value). \
+            order_by(desc(Filing.effective_date)). \
+            all()
```

Comment:
Just out of curiousity, does line 694 return all records in the filings table, or do the subsequent lines run on the db pod and therefore only the matching records are returned from the db?

Reply:
(Asking because I know the API and the db are in different pods, so the first case would be a lot of data bandwidth.)

Reply:
I believe it will return all incomplete filings for the specific business.

Reply:
Yes but is the complete list of filings transferred from the db to the Legal API over the network, or is the filtering done at the db?

Reply:
I see what you mean.  I'm sure it's done on the db level.  But will check.

Reply:
Confirmed that sqlalchemy constructs the sql and executes the query at the db level.

Reply:
Awesome. Makes sense; I wasn't sure :)